Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -40,7 +40,8 @@ class MainFrameMenu extends Atomic.ScriptObject {
         MenuItemSources.createMenuItemSource("menu tools", toolsItems);
         MenuItemSources.createMenuItemSource("menu developer", developerItems);
         MenuItemSources.createMenuItemSource("menu help", helpItems);
-
+        this.goScreenshot = 0;
+        this.subscribeToEvent(Atomic.UpdateEvent((ev) => this.handleScreenshot(ev)));
```

Comment:
Adding additional event handlers which are called per frame (especially scripted ones) can add up, I wonder if we can bring in the EventLoop stuff @shaddockh did on this, ```setTimeout``` seems like what we want here, which would also remove the need to the bookkeeping ```goScreenshot``` member, a couple examples:

Delayed scene load: https://github.com/AtomicGameEngine/AtomicExamples/blob/1f1bc5c8aa5676e3b084b874a7a5718fe0e3808f/SFXR/JavaScript/Resources/Scripts/main.js#L3

The events example itself: https://github.com/AtomicGameEngine/AtomicExamples/blob/master/EventLoop/JavaScript/Resources/Scripts/main.js

I looked and didn't see an example of the AtomicEventLoop module being brought into the editor's TS, I think the event module needs to be exposed, @shaddockh what do you think of adding the functions to duktape.d.ts, as they are kind of "built-ins"? https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/TypeScript/duktape.d.ts